README.md

```diff
@@ -1,77 +1,373 @@
+
+#### Print Working Directory 
+
+To help you know where you are currently in the Terminal we need to print the wordking directory. A directory would look like a folder on the screen of the GUI( Graphical User Interface). Type this command and hit enter:
 
-Below is a tutorial covering what web development is, what web development is comprised of, as well as how it all comes together to create
-web applications. The tutorial mentions installing Atom, don't worry about that. You're going to use VScode instead. 😉
+MacOS: `pwd`
+Windows: `cd`
 
-[HTML & CSS Tutorial (Go Through The Tutorial)](https://internetingishard.com/html-and-css/)
+#### List Directories
+To list all the directories and files in your current location type:
+MacOS : `ls`
+Windows: `dir`
 
-Getting a good grasp of how to layout websites is VERY important. The course below is going to get you moving on just that with some cool new additions to CSS.
+#### Make Directory
 
-<h2 id="javascript">JavaScript</h2>
+We want a place where we can put all our work in. While in the terminal make a new directory called VetsWhoCode by typing mkdir in Windows or MacOS and press enter:
 
-JavaScript is where you go from being an artist to being a wizard in web development. A deep knowledge of JavaScript fundamentals will make you a highly employable developer. Complete Javascript Info to get a good beginner grasp of JS.
+`mkdir VetsWhoCode`
 
-JavaScript is derived from ECMAScript. This is important since new versions come out having major impact on JavaScript. Read through this resource to understand the newest version of ECMAScript:
+#### Change Working Directory
 
+Now that we have made a new directory lets go there where we will put our work on our machine. te do that by in both windows and MacOS we do this by dyping cd then pressing enter:
 
+`cd VetsWhoCode`
 
-[Javascript Info](http://javascript.info/)
 
+#### Change to previous Directory
+If you want to go back to the previous directory for both Windows and MacOS you type this and press enter:
 
+`cd ..`
 
-<h2 id="capstone">Capstone</h2>
+#### Delete a directory
+Let's say you make a typo and need to delete a directory. Here are the commands to do that.
 
-Build a website telling about you and include your favorite things and then push it to GitHub. Bonus points if you use [GitHub Pages](https://pages.github.com/) or [surge.sh](https://surge.sh/) to host your site.
+MacOS; `rm -r NameOfDirectory`
+Windows: `rmdir /S NameOfDirectory`
+
+### Code Editor
+
+An Integrated Development Environment (IDE) is a software application that provides comprehensive tools and features to assist developers in writing, debugging, and deploying software. An IDE typically includes a code editor, a debugger, a compiler or interpreter, and other tools such as a profiler, a version control system, and a build automation tool.
```

Comment:
Should this list of features be a bulleted point list?

- code editor
- debugger
- compiler/interpreter
- profiler
- version control system
- build automation tool
- auto-completion
- syntax highlighting

Reply:
Since it's a natural language, I used commas. I can make it a list if easier to read.